### DISCUSSION TITLE
Multiple Harvester PC on different LAN/Network

### DISCUSSION THREAD

**Discussion by jackrohit001** [Chia-Network/chia-blockchain#8263] [General]
Hi, 

I am planning to start a new harvester on different location. 

In this case my plots will sync with each other or not?

**Comment by loppefaaret** [comment]
that should not be a problem - only if you got identical plots on the 2 locations, both location are farming the same data (the data that are identical in the 2 plots) and in essence wasting work if so... otherwise there should be nothing to worry about

**Comment by BrandtH22** [comment]
@jackrohit001 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
